sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/continuous/ContinuousExecution.scala

```diff
@@ -307,7 +311,8 @@ class ContinuousExecution(
     } catch {
       case t: Throwable if StreamExecution.isInterruptionException(t, sparkSession.sparkContext) &&
           state.get() == RECONFIGURING =>
-        logInfo(s"Query $id ignoring exception from reconfiguring: $t")
+        logInfo(log"Query ${MDC(QUERY_ID, id)} ignoring exception from reconfiguring: " +
+          log"${MDC(EXCEPTION, t)}")
```

Comment:
Usually we use `ERROR` for this one.

Reply:
Nice, I missed that we already had that log key, switched everything to use that instead.